cmsplugin_articles_ai/models/plugin_models.py

```diff
@@ -52,6 +52,17 @@ class ArticleListPlugin(CMSPlugin):
         verbose_name=_("tags"),
         related_name="article_list_plugins",
     )
+    exclude_categories = models.ManyToManyField(
+        Category,
+        verbose_name=("exclude categories"),
```

Comment:
Nice parenthesis, but perhaps you meant `_(...)` and not just `(...)`

Reply:
oh, heh, will fix